modules/developers-guide/pages/tutorials/reproducible-builds.adoc

```diff
@@ -0,0 +1,341 @@
+= Reproducible Canister Builds
+:IC: Internet Computer
+
+Thanks to its consensus protocol, the {IC} always runs canisters' code correctly.
+But this doesn't mean that it's running the _correct_ code for a canister.
+In particular, if you are a canister user as opposed to the owner, you likely want to verify that the canister is running some intended code before giving it control over your ICP tokens or confidential data.
```

Comment:
```suggestion
if you are using a canister somebody else developed, you may want to verify that the canister is indeed running some intended code before giving it control to make important decision for you, e.g., execute a canister smart contract involving ICP.
```

Reply:
At this stage, I don't want to suggest users should put confidential data on canisters.

Reply:
Makes sense